### ISSUE TITLE
Add clickable table of content per chapter for easier navigation

### ISSUE THREAD

**Issue by mikepan** [mikepan/GameEngineBook#18]

**Comment by MoritzBrueckner** [comment]
Is it ok for you when I start with the first chapter? For the other chapters I think I/we should wait for them to be finished

**Comment by mikepan** [comment]
yup of course. sounds good to me. We can probably also just defer this until all the chapters are done and generate them through jekyll.
